### ISSUE TITLE
[Tokenizer]: [A11y] Using Tokenizer for Deleting Filters - Clear All Button Is Not Accessible Via Keyboard

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#7207]
### Describe the bug

Actual Results

Clear all cannot be accessible via keyboard only operations

### Isolated Example

https://jvmiplyzwogithub-qyuo--5173--33edf5bb.local-credentialless.webcontainer.io/

### Reproduction steps

1.Go to https://jvmiplyzwogithub-qyuo--5173--33edf5bb.local-credentialless.webcontainer.io/
2. Using Tab key to get focus to the token item
3. The Clear All button could not be accessible
...

### Expected Behaviour

Expected Results 

Clear All button should be keyboard accessible

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP Successfactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @sleric1024 

the "fullscreen" link of StackBlitz examples is not sharable. Please share the URL starting with https://stackblitz.com instead.

**Comment by sleric1024** [comment]
@Lukas742 ,
Thanks for the information, https://stackblitz.com/edit/github-dr9gxkcb?file=src%2FApp.tsx
Can you try this link?

**Comment by Lukas742** [comment]
@sleric1024 yup, that works. Thanks :) 

I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
